packages/stream-metadata/src/node.ts

```diff
@@ -72,14 +73,22 @@ export function setupRoutes(srv: Server) {
 	/*
 	 * Routes
 	 */
-	srv.get('/health', checkHealth)
-	srv.get('/space/:spaceAddress', async (request, reply) => fetchSpaceMetadata(request, reply))
+
+	// cached
+	srv.get('/media/:mediaStreamId', fetchMedia)
+	srv.get('/user/:userId/image', fetchUserProfileImage)
 	srv.get('/space/:spaceAddress/image', fetchSpaceImage)
+
+	// not cached
+	srv.get('/health', checkHealth)
+
+	// should be cached, but not before implementing /refresh on metadata routes
+	srv.get('/space/:spaceAddress', fetchSpaceMetadata)
```

Comment:
We're already calling refresh for the `/space/:spaceAddress` when a space is updated (image, description, name)

Reply:
@miguel-nascimento i just pushed a commit. can you take a look again?